File: test/Scratch.java
public class Scratch {
  public String foo(String bar) {
    return bar;
  }

  public String hello(String bar) {
    return "Hello, " + bar + "!";
  }

  public static void main(String[] args) {
    System.out.println(new Scratch().hello("world"));
  }
}

File: test/scratch.py
#!/usr/bin/env python3

#
# Python scratch pad
#

from uuid import uuid4
from scratch_module import FOO

def foo(bar):
    print(f"foo executed with {bar}")
    return "value"


print(FOO)
print(uuid4())
bar = "hello"
foo(bar)
